README.md

```
@@ -1,61 +1,137 @@
+
+1. Fork the repository
+2. Create your feature branch (`git switch -c feature/<feature-name>`)
+3. Follow the [Git Flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) workflow
+4. Commit your changes (`git commit -m 'Adds some amazing feature'`)
+5. Push to the branch
+6. Open a Pull Request in your own repo
+
+
+## 📚 Learning Resources
+
+### Video Tutorials
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)